Library/Homebrew/brew.sh

```diff
@@ -584,6 +584,11 @@ then
   export HOMEBREW_BOTTLE_DOMAIN="$HOMEBREW_BOTTLE_DEFAULT_DOMAIN"
 fi
 
+if [[ -n "$HOMEBREW_DEVELOPER" || -n "$HOMEBREW_DEV_CMD_RUN" ]]
+then
+  export HOMEBREW_BOOTSNAP="1"
```

Comment:
You would have better performance if you rewrote brew in rust.

_(Don't hit me, I'm just kidding!)_

Reply:
Wait, `HOMEBREW_BOOTSNAP` doesn't run Homebrew in Rust? That's what I thought it did...

Reply:
I look forward to the day where we install `brew` to build `brew`.